blog/2021-06-29-transaction-presigning.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Pre-signing transactions for non-custodial blockchain protocols"
+author: thomas
+author_url: https://github.com/thomaseizinger
+author_image_url: https://avatars1.githubusercontent.com/u/5486389
+tags: [monero]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<!-- TODO: Find a good image -->
+<!-- <img alt="Blackboard" src={useBaseUrl('blog/assets/images/2021-06/monero-lesson.png')} /> -->
+
+Why you want it, why it doesn't work on present day Monero and what that means for applications on top.
```

Comment:
To make this timeless you could add a version of Monero here.

Reply:
Do you mean saying something like "doesn't work on Monero 0.17"?

But that doesn't sound as nice :sweat_smile:

Reply:
Add a footnote? :)

Reply:
Yes, that was my thinking :)